### PR TITLE
COMMERCE-600 Payment Engine -- source formatting

### PR DESCRIPTION
**Pull request by ethanbustad**
@marco-leo @luca-pellizzon

### CODE REVIEW

**Comment by ethanbustad** [comment]
ci:test

File: commerce-checkout-web/src/main/java/com/liferay/commerce/checkout/web/internal/display/context/util/CommerceCheckoutRequestHelper.java [hunk 1]
```diff
@@ -23,10 +23,8 @@
  */
 public class CommerceCheckoutRequestHelper extends BaseRequestHelper {
 
-	public CommerceCheckoutRequestHelper(
```
Comment (marco-leo) [inline commerce-checkout-web/src/main/java/com/liferay/commerce/checkout/web/internal/display/context/util/CommerceCheckoutRequestHelper.java]: @ethanbustad This is an old convention, HttpServletRequest httpServletRequest is the new one. Please revert this commit

**Review by marco-leo — CHANGES_REQUESTED** [review]

**Comment by ethanbustad** [comment]
ci:test
@luca-pellizzon made the changes based on our discussion on the other PR.
@marco-leo changed `HttpServletRequest` variables back to `httpServletRequest`.

**Comment by ethanbustad** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:test

**Pull request merged**